src/core.c/Supply-factories.pm6

```diff
@@ -76,8 +76,79 @@
         method sane(--> True) { }
         method serial(--> True) { }
     }
-    method interval(Supply:U: $interval, $delay = 0, :$scheduler = $*SCHEDULER) {
-        Supply.new(Interval.new(:$interval, :$delay, :$scheduler));
+
+    class IntervalBetween does Tappable {
+        has $!scheduler;
```

Comment:
Why not `has $!scheduler is built;` and 2 lines below instead of `submethod BUILD` ??

Reply:
For consistency with Interval class.